src/zc/relation/catalog.py

```diff
@@ -11,19 +11,19 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-import copy
-import sys
-
-import zope.interface
-import zope.interface.interfaces
-
 import BTrees
 import BTrees.check
 import BTrees.Length
+import copy
```

Comment:
This is inconsistent with all other projects. Please use the `isort` configuration generated by meta/config and move from there.